net/cloak/files/cloak.init

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2010-2011 OpenWrt.org
```

Comment:
Are you affiliated with OpenWrt anyhow?

Reply:
Not really. How do you suggest keep it? my own copyright?